packages/bundle/src/adaptiveCards/Attachment/CommonCard.js

```diff
@@ -1,42 +1,29 @@
 import { connectToWebChat } from 'botframework-webchat-component';
 import AdaptiveCardBuilder from './AdaptiveCardBuilder';
```

Comment:
Can you help reordering the imports?

I believe what we are doing is:
- Put NPM package imports first
- Give a blank line
- Put internal imports last

So, it's more like.

```js
import React from 'react';
import ReactDOM from 'react-dom';

import App from './App';
import SomeUIComponent from './SomeUIComponent';
```